db/books.cds

```diff
@@ -8,6 +8,11 @@ using {
     cuid
 } from '@sap/cds/common';
 
+entity Csv @odata.singleton {
```

Comment:
As already suggested earlier: We don't really want a database persistence for this. Conceptually this better fits as an entity that is directly defined in the admin service and is annotated with @cds.persistence.skip to prevent any database table to be created for it.

Reply:
That's a great option that I was not aware of. Thank you for the suggestion.